app/Http/Controllers/Admin/CategoryController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Models\Category;
+use App\Http\Requests\Category\StoreCategoryRequest;
+use App\Http\Requests\Category\UpdateCategoryRequest;
+use App\Http\Resources\CategoryResource;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\Response;
+use App\Http\Controllers\Controller;
+
+
+
+class CategoryController extends Controller
+{
+    public function __construct() 
+    {
+        $this->middleware('auth:api', ['except' => ['index', 'show']]);
```

Comment:
this is better done as  a middleware then make a route group that uses it so you don't have to repeat the same logic over and over with every controller